lava/lava-job-definitions/testplans/helloworld-update-pelion.yaml

```diff
@@ -13,13 +13,15 @@
 
     {{ macros.avahi_discovery() | indent }}
```

Comment:
Looks like we should create a task to in future roll these different set up steps into some other macro - there by reducing the duplication in all these tests.

Reply:
Created IOTMBL-2463 so this gets picked up in future.